packages/dimen/image.ts

```diff
@@ -0,0 +1,2 @@
+export const mediumImage = 0.2
```

Comment:
This belongs under `theme/include/dimensions.ts`

- don't abbreviate words
- stuff related to visuals goes under theme
- primitive exports are in theme/includes